component.js

```diff
@@ -274,7 +286,7 @@ function factory (initialOptions) {
       }
 
       return create;
-    };
+    }
```

Comment:
Very nit-picky, but should be a semi-colon to conform to the rest of the code style.